solutions/agents/variables.tf

```diff
@@ -254,3 +254,121 @@ variable "cloud_monitoring_agent_tolerations" {
       key : "node-role.kubernetes.io/master"
   }]
 }
+
+##############################################################################
+# Logs Agents variables
+##############################################################################
+
```

Comment:
please set `log_analysis_enabled` to false now. And for all the variable descriptions for log analysis, please prefix them with `DEPRECATED: ` like I did in the module.